esp/bindings/SOAP/Platform/soapbind.hpp

```diff
@@ -125,14 +125,17 @@ class esp_http_decl CSoapResponseBinding : public CSoapComplexType,
 
 class esp_http_decl CSoapRequestBinding : public CSoapComplexType,
     implements IRpcRequestBinding,
-    implements IEspRequest
+    implements IEspRequest,
+    implements IEspClientRpcSettings
 {
 private:
     StringBuffer url_;
     StringBuffer proxy_;
     StringBuffer userid_;
     StringBuffer password_;
     StringBuffer realm_;
+    unsigned connectTimeoutMs = 0;
```

Comment:
This may prompt me to refactor the names in this entire file (probably a good thing), but for now I will change it to match.